app/src/main/java/com/winlator/contents/ContentsManager.java

```diff
@@ -206,6 +210,13 @@ public void extraContentFile(Uri uri, OnInstallFinishedCallback callback) {
     }
 
     public void finishInstallContent(ContentProfile profile, OnInstallFinishedCallback callback) {
+        // Check if a version with this name already exists and find next available version code
+        int nextVerCode = findNextAvailableVersionCode(profile.verName, profile.type);
+        if (nextVerCode > 0) {
+            // A version with this name exists, use incremented version code
+            profile.verCode = nextVerCode;
+        }
+
```

Comment:
This is required if the verCode in the profile is not 0. 

A small snag that came up with a couple of dists.